contrib/tkn-image/Dockerfile

```diff
@@ -2,9 +2,10 @@ ARG GOLANG_VERSION=1.13.8
 ARG DEBIAN_VERSION=10
 
 FROM golang:${GOLANG_VERSION} as builder
+ARG RELEASE_VERSION=
```

Comment:
Should we have latest as default?

Reply:
I'm letting the go code handling that. right now, it's setting it to `dev` if the given.